tornado_json/schema.py

```diff
@@ -158,9 +151,8 @@ def _wrapper(self, *args, **kwargs):
             setattr(self, "body", input_)
             # Call the requesthandler method
             output = rh_method(self, *args, **kwargs)
-            # If the rh_method returned a Future a la `raise Return(value)`
-            #   we grab the output.
-            if is_future(output):
+            #   or a python 3 coroutine we grab the output.
```

Comment:
Is this correct? This comment seems to start suddenly with an "or"; maybe you mistakenly deleted part of the original comment?